src/go/rpk/pkg/tuners/network/nics.go

```diff
@@ -33,15 +33,15 @@ func getDefaultMode(
 		if err != nil {
 			return "", err
 		}
-		zap.L().Sugar().Debugf("Considering '%d' cores and '%d' PUs", numOfCores, numOfPUs)
 
-		if numOfPUs <= 4 || rxQueuesCount == int(numOfPUs) {
-			return irq.Mq, nil
-		} else if numOfCores <= 4 {
-			return irq.Sq, nil
-		} else {
-			return irq.SqSplit, nil
-		}
+		// Currently we use only the mq mode because the idea behind sq and sq-split modes is that
+		// a core is *dedicated* to IRQs (i.e., Redpanda core does not run on that core or lcore),
+		// but we don't currently support propagating a cpuset to Redpanda, so it will run on the IRQ
+		// core, causing it to be the primary bottleneck.
+		zap.L().Sugar().Debugf("Using mq mode (hardcoded) for '%d' cores, '%d' PUs and '%d' rx queues",
+			numOfCores, numOfPUs, rxQueuesCount)
+
+		return irq.Mq, nil
```

Comment:
In principle, this whole method could just be replaced by `return irq.Mq, nil` but I decided to leave most of the logic intact as we do plan to revisit sq modes at some point in the future, and we get some error handling benefits (?) in case the system layout is not as expected.

Reply:
Sounds good, maybe remove the log line in line 27 as well though. 

Might be slightly confusing as we are not calculating anything anymore.

Reply:
Good idea, line 27 is gone and I've added the NIC name to the remaining log message.